doc/rtd/reference/datasources/openstack.rst

```diff
@@ -125,4 +125,35 @@ path; if found, settings are applied after (and, hence, overriding) the
 settings from static vendor data. Both sets of vendor data can be overridden
 by user data.
 
+.. _datasource_ironic:
+
+OpenStack Ironic Bare Metal
+===========================
+During boot, cloud-init typically has to identify which platform it is running
+on. Since OpenStack Ironic bare metal doesn't provide a method for cloud-init
+to discover that it is running on Ironic, extra user configuration is required.
+
+Cloud-init provides two method to do this:
+
+Method 1:
```

Comment:
Having "method 1" and "method 2" as the headers is not ideal, as when it comes up in the RHS navigation menu it's not very descriptive. I'd recommend that instead, we make these into a numbered list.

Reply:
Agreed, the navigation menu with this state is not ideal. I'm having a hard time picturing a numbered list looking good, however, since each method has some text and an example in it. Could you share an example of what you mean?

Alternatively I could improve the navigation by adding context to the title:
```
Method 1: Configuration file
...
Method 2: Kernel commandline
```